api/v1/models/user_model.py

```diff
@@ -0,0 +1,28 @@
+"""USER ENTITY """
+import uuid
+from werkzeug.security import generate_password_hash
+USERS = []
+
+
+class Users:
```

Comment:
@Busobozihakiim,  nice work he, however, The user class should have all the methods that deal with a user, for instance, creating a user, getting a user or deleting a user. we should put all the validation in a validation file so we just call them in other files when we need to use them

Reply:
the validation file does exist, and in this Pr, the user class currently  has methods of creating a user and checking if  the email they use for login exists

Reply:
@Busobozihakiim , my mistake..I was looking at someone else's PR..